### PR TITLE
installing Python 3.6 on Centos

### PR DESCRIPTION
**Pull request by keichwa**
# Description

Added information about installing Python 3.6 on Centos in the Client Configuration Guide (bsc#1187335).
https://github.com/SUSE/spacewalk/issues/15200

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

https://github.com/SUSE/spacewalk/issues/15200

### CODE REVIEW

File: modules/client-configuration/pages/snippets/appstream_admon.adoc [hunk 1]
```diff
@@ -5,6 +5,8 @@ This results in the {productname} {webui} showing incorrect package information.
 You cannot perform package operations such as installing or upgrading directly from modular repositories using the {webui} or API.
 
 You can use the AppStream filter with content lifecycle management (CLM) to transform modular repositories into regular repositories.
+Make sure to include ``python:3.6`` using an AppStream filter.
```
Comment (mcalmer) [inline modules/client-configuration/pages/snippets/appstream_admon.adoc:8]: ... if you want to use `spacecmd` on the clients.

In most cases people do not need spacecmd and if this is the case, they can also use other python versions.

File: modules/client-configuration/pages/clients-centos.adoc [hunk 2]
```diff
@@ -112,6 +112,9 @@ If you do not add both channels, you cannot create the bootstrap repository, due
 ====
 endif::[]
 
+If you are using modular channels, you must enable the Python 3.6 module stream on the client.
+If you do not provide Python 3.6, the installation of the [package]``spacecmd`` package will fail.
+
```
Comment (mcalmer) [inline modules/client-configuration/pages/clients-centos.adoc:117]: This is true for all RHEL 8 like OSes. The page is named clients-centos.adoc. Is this re-used also fir alma, OL8, Rocky and Native RedHat?

**Review by cbbayburt — APPROVED** [review]

File: modules/client-configuration/pages/clients-centos.adoc (hunk 2 above)
Comment (keichwa) [reply modules/client-configuration/pages/clients-centos.adoc:117]: > This is true for all RHEL 8 like OSes. The page is named clients-centos.adoc. Is this re-used also fir alma, OL8, Rocky and Native RedHat?

Now we cover: Centos, Oracle Linux, Almalinux, Rocky Linux, SUSE Linux Enterprise Server with Expanded Support, and Red Hat.  SLESES was missing, I just added it.  I hope that that's ok.

**Comment by keichwa** [comment]
@cbbayburt I added SLES ES.  Is this ok?  Please review once again.

**Review by cbbayburt — APPROVED** [review]

**Pull request merged**